package-manifest.toml

```diff
@@ -61,11 +61,11 @@ output.type = "zone"
 setup_hint = "Run `./tools/ci_download_cockroachdb` to download the necessary binaries"
 
 [package.internal-dns]
-service_name = "internal-dns"
+service_name = "internal_dns"
```

Comment:
This was basically just a result of turning the name into an enum; the tests got mad at me when I was serializing the name as "internal-dns" instead of "internal_dns".